gridfs/grid_file.py

```diff
@@ -679,10 +707,21 @@ def close(self):
         if self.__chunk_iter:
             self.__chunk_iter.close()
             self.__chunk_iter = None
+            self.__closed = True
```

Comment:
I think we need a super().close() call here now.

Reply:
Fixed